build/openwisp_dashboard/load_init_data.py

```diff
@@ -80,7 +80,7 @@ def create_default_cert(vpnOrg, defaultCa, x509NameCert):
         defaultCert.full_clean()
         defaultCert.save()
         return defaultCert
-    return Cert.objects.get(name=x509NameCert)
+    return Cert.objects.filter(name=x509NameCert).distinct()
```

Comment:
I don't think this fixes the problem, it only applies a patch to hide the error. I don't think it's good for us to hide the error. I'd rather want to fix the core of the issue, 2 certificates shouldn't be created in the first place.
It only happens sometimes but it's a disaster when it happens really!

Regarding the travis failure: 
Please check the travis logs, I see: 
```
- missing prefix in the commit short description

  Eg: "[module/file/feature] Action performed"

Please read our guidelines at: http://openwisp.io/docs/developer/contributing.html#commit-message-style-guidelines

ERROR: Commit name check failed!
```